packages/main/src/Menu.ts

```diff
@@ -461,22 +478,32 @@ class Menu extends UI5Element {
 	}
 
 	_createSubMenu(item: MenuItem, opener: HTMLElement) {
-		if (item._subMenu) {
-			return;
+		let subMenu = item._subMenu;
+		let subMenuRef = this._getSubmenuReference(item);
+
+		if ((!subMenu && !this._isSubMenu) || (!subMenuRef && this._isSubMenu)) {
+			const ctor = this.constructor as typeof Menu;
```

Comment:
const ctor = this.constructor as typeof Menu;
subMenu = document.createElement(ctor.getMetadata().getTag()) as Menu;
can be replaced with
subMenu = document.createElement(this.tagName.toLowerCase()) as Menu;

Or this.tagName.toLowerCase() can be extracted in variable, as I see it used in few times in the method.

Reply:
done